Server/src/lobbycoordinator.erl

```diff
@@ -0,0 +1,100 @@
+-module(lobbycoordinator).
+-export([init/0]).
+%%Version 1.0
+%%Authors: Sebastian Fransson, Tim Jonasson
+
+%Initializes the lobby
+init() -> 
+  loop([], 0).
+  
+%the loop keeps track of the rooms and handles lobby requests.
+loop(Rooms, Lobby_increment) ->
+  receive 
+    %Creates a lobby and saves the monitor reference as well as lobby info to the list of rooms.
+    {Creator_Socket, create_lobby, Password} -> 
+	  % Spawn the process and save the Pid.
+	  Pid = spawn(fun () -> lobby:init(Creator_Socket, Password, Lobby_increment) end),
+	  % Save the reference so that we can use it later for demonitoring.
+      Ref = monitor(process, Pid),
+	  % Send a confimation to the client saying that the lobby was created.
+	  gen_tcp:send(Creator_Socket, io_lib:format("INFO# Created lobby with id:,  ~p", [Lobby_increment]) ++ "~"),
+	  loop([{Lobby_increment, Password, Pid, Creator_Socket, Ref}|Rooms], Lobby_increment + 1);
+	
+	%Demonitors, kills and removes the host's lobby from the list of rooms.
+	{Creator_Socket, remove_lobby} -> 
```

Comment:
I will just assume that this functionality works.
Because there are 4 lines in a row doing list stuff.